codeworld-compiler/codeworld-compiler.cabal

```diff
@@ -0,0 +1,62 @@
+Name:                codeworld-compiler
+Version:             0.1.0.0
+Synopsis:            ghcjs based standalone compiler for codeworld
+License:             Apache-2.0
+License-file:        LICENSE
+Author:              The CodeWorld Authors
+Maintainer:          pranjaltale16@gmail.com
```

Comment:
Are you volunteering? :)  If so, standard format here would be:

    Maintainer:      Pranjal Tale <pranjaltale16@gmail.com>